internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -54,9 +54,10 @@ type BundleReconciler struct {
 	Unpacker   source.Unpacker
 }
 
-//+kubebuilder:rbac:groups=core.rukpak.io,resources=bundles,verbs=list;watch
+//+kubebuilder:rbac:groups=core.rukpak.io,resources=bundles,verbs=list;watch;update;patch
```

Comment:
Any idea on why this is needed given we should only need to perform updates/patches on the status sub-resource?

Reply:
This is needed to add finalizers. We haven't needed it before because the plain provisioner has */*/* in support of the bundle instance side of things.

I noticed we needed it in #387 for the registry provisioner, but this is unrelated to this PR, so I'll pull this out and keep it in #387

Reply:
ah right forgot about the finalizer aspect here. good catch.